pkg/virt-api/webhooks/validating-webhook/admitters/vmclone-admitter.go

```diff
@@ -226,6 +230,29 @@ func validateSource(client kubecli.KubevirtClient, vmClone *clonev1alpha1.Virtua
 	return causes
 }
 
+func validateTarget(vmClone *clonev1alpha1.VirtualMachineClone) []metav1.StatusCause {
+	var causes []metav1.StatusCause
+
+	const virtualMachineKind = "VirtualMachine"
```

Comment:
nit: I see `VirtualMachine` and `VirtualMachineSnapshot` as plain strings too many times in this package.
Maybe for the future, it is worth it to define some constants?

Reply:
I agree.
Will follow up on that.